### DISCUSSION TITLE
I am unable to become a part of GitHub education despite studying comp sci at uni.

### DISCUSSION THREAD

**Discussion by jayco10125** [community/community#49611] [GitHub Education] [answered]
I have my student id, my enrolment advice form, my enrolment tab in the uni portal, and my timetable for THIS semester dated with my name as well as my personal details in the uni portal but it will not even process because it says that my email is not verified even though I have checked that it is completely verified and I am using my uni email. My other friends at the same uni are having the same problem and cannot access GitHub education.

Essentially when I try to apply it says "attendees of this school must verify their academic email address". I assume that means using it as my primary email then going to my email and hitting the verify button that they emailed. I did that. It says verified.

I am unsure what else I can do and any tickets through support say to ask the community or are ignored.

**Comment by Abinashbunty** [comment, marked as answer]
Is your university email address already verified on your GitHub profile? After that is verified, re-apply.

**Reply by jayco10125** [reply]
It was already verified and have since re-applied twice, thanks for the reply though!

**Reply by Abinashbunty** [reply]
Perhaps make sure that the academic enrolment duration is explicitly mentioned in your enrolment document. And also the fact that it is in line with your expected graduation date.

**Reply by jayco10125** [reply]
just tried again and it came up with this: 
<img width="766" alt="Screen Shot 2023-03-09 at 7 26 58 pm" src="https://user-images.githubusercontent.com/47494262/224010246-70a1d6c6-a061-473c-90c0-9ff997f60f3c.png">

**Reply by Abinashbunty** [reply]
I saw the list of schools from GitHub [here](https://education.github.com/schools). Your school is not included at the moment it seems. So you might have to use your personal email address because GitHub won't be able to verify your school email address associated with your school.

**Reply by jayco10125** [reply]
does being in the list here not mean its included? 
<img width="1512" alt="Screen Shot 2023-03-09 at 7 43 33 pm" src="https://user-images.githubusercontent.com/47494262/224013530-8e9aa8a7-85cc-42dc-b5db-2b4287fb37db.png">

**Reply by Abinashbunty** [reply]
The list perhaps includes a general list of schools and universities worldwide. But not necessarily all are partnered with GitHub. 

![image](https://user-images.githubusercontent.com/25522048/224014997-5bb017c5-a7dc-4d8b-bd17-665410c62388.png)

You can see here in the map that your school has not yet partnered with GitHub perhaps.

**Reply by jayco10125** [reply]
yeah its not on the map, so can I still be a part of github education if my uni has not partnered? and if so how can I join?

**Reply by Abinashbunty** [reply]
You can inform your university helpdesk or service desk who are in charge of all the IT related stuff. You can let them know about this and perhaps send them the link that I shared before. This one: [GitHub Education](https://education.github.com/schools). 😄 They can carry it forward from there.

**Reply by Abinashbunty** [reply]
> You can inform your university helpdesk or service desk who are in charge of all the IT related stuff. You can let them know about this and perhaps send them the link that I shared before. This one: [GitHub Education](https://education.github.com/schools). 😄 They can carry it forward from there.

On another note, if I have guided you more or less in the right direction, you can approve my reply as an answer for this discussion. 😄 Thanks in advance.
